cmd/textile/main.go

```diff
@@ -61,6 +61,7 @@ var (
 	cmdTimeout     = time.Second * 10
 	loginTimeout   = time.Minute * 3
 	addFileTimeout = time.Minute
+	getFileTimeout = time.Minute
```

Comment:
Dunno what this should really be. Maybe no timeout? User should be able to download massive files. Same goes for `addFileTimeout`.

Reply:
Yea seems like something really high like a day or three or no timeout.

Reply:
Cool, I'll make them a day for now.